### ISSUE TITLE
[Bug] "Cannot create a file when that file already exists" when upgrading texlive

### ISSUE THREAD

**Issue by weeix** [naveen521kk/chocolatey-texlive#6]
### Package Version

v2022.20220302 -> 2022.20220402

### Current Behaviour

```
Chocolatey v1.1.0
Upgrading the following packages:
texlive
By upgrading, you accept licenses for the packages.

You have texlive v2022.20220302 installed. Version 2022.20220402 is available based on your source(s).
Progress: Downloading texlive 2022.20220402... 100%

texlive v2022.20220402 [Approved]
texlive package files upgrade completed. Performing other installation steps.
Writing Profile
Writing Profile using Passed Parameters.
TeX Live Profile will be written to C:\tools\texlive.profile
Extracting C:\ProgramData\chocolatey\lib\texlive\tools\install-tl.zip to C:\tools\temp-install-tl-050422...
C:\tools\temp-install-tl-050422
ERROR: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Cannot create a file when that file already exists.

The upgrade of texlive was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\texlive\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey upgraded 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - texlive (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\texlive\tools\chocolateyinstall.ps1'.
 See log for details.
```

### Expected Behaviour

Upgrades successfully.

### Steps To Reproduce

1. Run `choco upgrade -y texlive`
2. See error `ERROR: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Cannot create a file when that file already exists.`

### Environment

```markdown
- Operating System: Microsoft Windows NT 10.0.22000.0
- PowerShell Version: 5.1.22000.282
- Shell: PowerShell
```

### Chocolatey Log

```shell
https://gist.github.com/weeix/7b55f85d3601c262e451eb2127205a4f
```

**Comment by dbourrion** [comment]
Hi
Same here with Windows 10

**Comment by naveen521kk** [comment]
I think https://github.com/naveen521kk/chocolatey-texlive/commit/615e177988c4785d8c48b9f8e42d0920a1c96465 should fix this issue, I'll wait for the CI to pass and then push the version on chocolatey for you to test.

**Comment by naveen521kk** [comment]
I've pushed https://community.chocolatey.org/packages/texlive/2022.20220405 can you check if it works as expected?

**Comment by dbourrion** [comment]
Hi
Not working here 

texlive v2022.20220302 (forced) [Approved]
texlive package files install completed. Performing other installation steps.
Writing Profile
Writing Profile using Passed Parameters.
TeX Live Profile will be written to C:\tools\texlive.profile
Extracting C:\ProgramData\chocolatey\lib\texlive\tools\install-tl.zip to C:\tools\temp-install-tl-050422...
C:\tools\temp-install-tl-050422
Automated TeX Live installation using profile: C:/tools/texlive.profile
Loading https://ctan.crest.fr/tex-archive/systems/texlive/tlnet/tlpkg/texlive.tlpdb
ERROR: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: System.Management.Automation.RemoteException
The install of texlive was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\texlive\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - texlive (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\texlive\tools\chocolateyinstall.ps1'.
 See log for details.
 
 -----
 
 perhaps a local problem on my computer

**Comment by dbourrion** [comment]
Oh, https://community.chocolatey.org/packages/texlive/2022.20220405 is working
Many thanks !!

**Comment by weeix** [comment]
Thanks for the quick fix! Version [2022.20220405](https://community.chocolatey.org/packages/texlive/2022.20220405) is now live, so I'm closing this issue.
